### ISSUE TITLE
Taking a snapshot of an iframe with scroll bars

### ISSUE THREAD

**Issue by richard919** [assertthat/selenium-shutterbug#30]
Hello,
I'm trying to take a snapshot of an iframe element that has a scrollbar.  Does anyone know of a way to do this that would be less invasive than creating a forked version of selenium-shutterbug with a new function similar to Browser.takeScreenshotEntirePage() that would scroll through an iframe and create an image of the whole iframe?

I'm using shutterbug as a part of Fitnesse/Xebium/Selenium in order to do regression testing, so I don't have access to the webpage that I'm taking a screenshot of (so I can't get rid of the scroll bars).  I'm not going to post a picture of the system I'm testing (I don't think the person who owns that system would be alright with that), but to help give an idea of what I'm talking about, the iframe has scroll bars similar to the code box in the following screenshot:

![screenshot_6](https://user-images.githubusercontent.com/17017180/30768618-c5123676-9fcf-11e7-9bca-21eb7d8a6952.png)

**Comment by glib-briia** [comment]
Hi @richard919 ,

Currently shutterbug doesn't support scrolling withing iframes. I'll take a look at possible options and get back to asap.

Thanks,
Glib

**Comment by glib-briia** [comment]
Implemented in v1.0 e.g
```
Shutterbug.shootFrame(driver, driver.findElement(By.id("myFrame")),
                Capture.FULL_SCROLL).save();
```
